ci-operator/step-registry/observers/resource-watch/observers-resource-watch-commands.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+export REPOSITORY_PATH="${ARTIFACT_DIR}/resource-watch-store"
```

Comment:
It's neat that we still get the git repo even if the signal is lost and we fail to tar it up.